theano/gof/theano_mod_helper.h

```diff
@@ -0,0 +1,28 @@
+#ifndef THEANO_MOD_HELPER
+#define THEANO_MOD_HELPER
+
+#include <Python.h>
+
+#ifndef _WIN32
+#define MOD_PUBLIC __attribute__((visibility ("default")))
+#else
+#define MOD_PUBLIC
```

Comment:
We may need to add DllExport (or Import) here, according to https://gcc.gnu.org/wiki/Visibility